hackathon_site/review/test_admin.py

```diff
@@ -98,14 +98,14 @@ def test_list_page_shows_submission_date(self):
         The submission date for a team is the date of the most recently submitted
```

Comment:
You could change this comment

Reply:
Eh, depends how "submitted" is interpreted. Should be clarified in the template repo, I'll leave it as is here